src/components/molecules/Candidate/Candidate.tsx

```diff
@@ -0,0 +1,53 @@
+import { ArrowBack} from "@mui/icons-material"
+import { Stack,Grid } from "@mui/material"
+import { useEffect, useState } from "react"
+import { currentdata, getCurrentdata } from "../../../Services/services"
+import ButtonComponent from "../../atoms/Button/ButtonComponent"
+import Text from "../../atoms/Typography/Text"
+import CandidateInfo from "../CandidateInfo/CandidateInfo"
+import Count from "../Countsearches/Count"
+import ReportInfo from "../ReportInfo/ReportInfo"
+
+
+
+const Candidate  =(id :any)=>{
+    const [currentdata,setCurrentData] = useState<currentdata[]>([])
```

Comment:
Molecules shouldn't contain any states or functions. Move this to organism or page containing this component